src/tests/test_cubicspline.cc

```diff
@@ -72,6 +72,32 @@ BOOST_AUTO_TEST_CASE(cubicspline_fit_test) {
   
 }
 
+BOOST_AUTO_TEST_CASE(cubicspline_matrix_test) {
+    
+  CubicSpline cspline;   
+  cspline.setBCInt(0);
+  cspline.GenerateGrid(0.4,0.6,0.1);
+  Eigen::MatrixXd A=Eigen::MatrixXd::Zero(1,6);
+  Eigen::MatrixXd Aref=Eigen::MatrixXd::Zero(1,6);
+  
+  Aref(0,0)=0.0;
```

Comment:
instead of this you can simply write `Aref<<0.0,-9.0,10.0,0.0,-0.3333333,-0.0166667;`

but it is fine this way too.

Reply:
Thanks